src/boss/cloudvm/local_worker.go

```diff
@@ -6,18 +6,22 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 
 	"github.com/open-lambda/open-lambda/ol/common"
 )
 
 // WORKER IMPLEMENTATION: LocalWorker
 type LocalWorkerPoolPlatform struct {
-	// no platform specific attributes
+	nextWorkerPort int
 }
 
 func NewLocalWorkerPool() *WorkerPool {
+	startPort, _ := strconv.Atoi(GetLocalPlatformConfigDefaults().Worker_Starting_Port)
```

Comment:
Shouldn't we pull from the actual config (https://github.com/open-lambda/open-lambda/blob/main/src/boss/config.go#L14) instead of the default?